pkg/controller/actuator_reconcile.go

```diff
@@ -49,48 +47,42 @@ func (a *actuator) Reconcile(ctx context.Context, log logr.Logger, cr *extension
 		return fmt.Errorf("could not create chart renderer for shoot '%s', %w", cr.Namespace, err)
 	}
 
-	// if the managed resource containing the prerequisites is not available yet, create it.
-	mr := &resourcesv1alpha1.ManagedResource{}
-	if err := a.client.Get(ctx, kutil.Key(cr.Namespace, GVisorManagedResourceName), mr); err != nil {
-		if !apierrors.IsNotFound(err) {
-			return err
-		}
```

Comment:
Why was this part removed? 👀

Reply:
Here, we reconcile the managedresource only if the Get fails, but we want to reconcile for the change in chart values as well.